content/docs/faq-functions.md

```diff
@@ -178,9 +178,9 @@ class Alphabet extends React.Component {
 }
 ```
 
-#### Nümunə: Arqumentlərin data-atributlarına göndərilməsi {#example-passing-params-using-data-attributes}
+#### Nümunə: Arqumentləri məlumat atributlarını isitifadə edərək göndərmək {#example-passing-params-using-data-attributes}
```

Comment:
Burada "data-atributları" sözünü saxlayın çünki burada HTML-in `data-*` atributlarından söhbət gedir. Məlumat atributları termini düzgün olsa da nədən danışıldığı bilinmir.